### PR TITLE
Split up the mean from the deviations to make the code a little easie…

### PR DESCRIPTION
**Pull request by samtruesdell**
…r to read.  Modified the plot and legend.

### CODE REVIEW

File: exampleAerialWeirCorrelation.R [hunk 1]
```diff
@@ -4,7 +4,6 @@
 # deviations, the correlation between the surveys
 # and the number of samples
 # 
-# test
```
Comment (samtruesdell) [inline exampleAerialWeirCorrelation.R:7]: Removed this line

File: exampleAerialWeirCorrelation.R [hunk 2]
```diff
@@ -13,36 +12,50 @@ n <- 50
 
 
 
+library(mvtnorm)
```
Comment (samtruesdell) [inline exampleAerialWeirCorrelation.R:15]: had to add the lib...

File: exampleAerialWeirCorrelation.R [hunk 3]
```diff
@@ -13,36 +12,50 @@ n <- 50
...
 
+# new trend for the indices
+x2 <- x1 + x0
 
+# graph paper setup
 hl <- seq(100, 400, 10)
 vl <- seq(-10, n+0.1*n, 1)
 
-plot(x0[,1], type='n', ylim=range(x0,x1))
+# plot the true mean and the deviations
```
Comment (samtruesdell) [inline exampleAerialWeirCorrelation.R:37]: Another test comment

File: exampleAerialWeirCorrelation.R (hunk 1 above)
Comment (cbelinsky) [reply exampleAerialWeirCorrelation.R:7]: Yo

Comment (cbelinsky) [reply exampleAerialWeirCorrelation.R:7]: I like guanacos

**Review by cbelinsky — DISMISSED** [review]
How about now?

**Review by samtruesdell — COMMENTED** [review]
done

**Pull request merged**